README.md

```diff
@@ -0,0 +1,113 @@
+##### Lab Orders
+* Supports adding of lab orders for patients.
+* Allow a user to toggle the urgency of a lab order.
+* Allows a user to view all the lab orders ordered for a patient
+# What This Module Does Not Do
+* Currently, only Drug and Lab orders are supported.
+* Does not support re-activating a discontinued drug order.
+  
+# Download
+> To view or download the OpenMRS Order Entry UI Module, please go to the module repository here:
+
+* To download module: https://addons.openmrs.org/#/show/org.openmrs.owa.orderentryui 
+* To view or download source code:  https://github.com/openmrs/openmrs-owa-orderentry
+# Video Tutorial
+A video tutorial walkthrough of the Order UI can be watched here : https://www.youtube.com/watch?v=7lhLOwZLqzo
+
+# Resources
+* [Order Entry UI End User Guide](https://wiki.openmrs.org/display/projects/Order+Entry+UI+End+User+Guide)
+* [Order Entry UI Administrator Guide](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Administrator+Guide)
+* [GitHub project (source code)](https://github.com/openmrs/openmrs-owa-orderentry)
+* Order Entry UI Release Note
+* [Sprint 1](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+1)
+* [Sprint 2](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+2)
+* [Sprint 3](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+3)
+* [Sprint 4](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+4)
+* [Sprint 5](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+5)
+* [Sprint 6](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+6)
+* [Sprint 7](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+7)
+* [Sprint 8](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+8)
+* [Sprint 9](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+9)
+* [Sprint 10](https://wiki.openmrs.org/display/projects/Order+Entry+UI+Sprint+10)
+
+# Team Lead
+
+[Daniel Kayiwa](https://wiki.openmrs.org/display/~dkayiwa)
+
+# Developers
+* [Flavia Nshemerirwe](https://wiki.openmrs.org/display/~flavia)
+* [Fredrick Mgbeoma](https://wiki.openmrs.org/display/~fred)
+* [Geoffrey Asiimwe](https://wiki.openmrs.org/display/~geofrocker)
+* [Osaze Edo-Osagie](https://wiki.openmrs.org/display/~zeze)
+* [Betty Kebenei](https://wiki.openmrs.org/display/~betty)
+* [Lanre Lawal](https://wiki.openmrs.org/display/~larrystone)
+* [Gabriel Micah](https://wiki.openmrs.org/display/~topseysuave)
+* [Efosa okpugie](https://wiki.openmrs.org/display/~efosa)
+* [Tunmise ogunniyi](https://wiki.openmrs.org/display/~tunmi)
+* [Isaiah Rotimi Yemitan](https://wiki.openmrs.org/display/~rotimi)
+* [Daramola Ajiboye](https://wiki.openmrs.org/display/~daramola98)
+* [Korir Desmond](https://wiki.openmrs.org/display/~desmond)
+* [Titus Kipkemboi](https://wiki.openmrs.org/display/~tittoh)
+
+## Issues
+
+If you want help fix existing issues or you found a bug and want to tell us please go to
+
+https://issues.openmrs.org
```

Comment:
Better to mention the direct project link here

Reply:
what's the link ?